scripts/lib/build-pgroonga

```diff
@@ -2,8 +2,13 @@
 set -x
 set -e
 
-PGROONGA_VERSION="2.1.6"
+release=$(lsb_release -sc)
 
+if [ "$release" = "buster" ]; then
+	PGROONGA_VERSION="2.1.8"
+else
+	PGROONGA_VERSION="2.1.6"
+fi
```

Comment:
What's the story here - is 2.1.8 needed to install on buster?  If so, I think we should just use 2.1.8 unconditionally, and add a comment that we've pinned the version for predictability, but we generally want to be using a recent version.

Reply:
Also the whitespace here is weird -- please use 4 spaces in all shell scripts.

Reply:
> Also the whitespace here is weird -- please use 4 spaces in all shell scripts.

Sorry about that. I must have missed that

> What's the story here - is 2.1.8 needed to install on buster? If so, I think we should just use 2.1.8 unconditionally, and add a comment that we've pinned the version for predictability, but we generally want to be using a recent version.

I wanted to ask you about that. 2.1.8 is required to install on buster(because of postgresql 11) but I was not sure if I should have totally replaced 2.1.6